spring-kafka/src/main/java/org/springframework/kafka/listener/AbstractMessageListenerContainer.java

```diff
@@ -619,6 +623,11 @@ public final void stop(boolean wait) {
 		}
 	}
 
+	@Override
+	public void enforceRebalance() {
+		this.enforceRebalanceRequested.set(true);
```

Comment:
I think we have to leave it here without any implementation.
The point is that we can simply set the value in that `protected final` from target implementations.